roles/docker/templates/docker-svc.j2

```diff
@@ -12,7 +12,7 @@ ExecStart=/usr/bin/docker daemon --storage-opt dm.thinpooldev=/dev/mapper/contiv
 {% else %}
 ExecStart=/usr/bin/docker daemon -H fd:// --cluster-store=etcd://localhost:{{ etcd_client_port1 }}
 {% endif %}
-MountFlags=slave
+MountFlags=shared
```

Comment:
I should clarify that -- we *need* shared, but for those who don't (e.g., netplugin-only consumers on swarm) probably don't want it.